### PR TITLE
Throw error if the specified project directory is part of existing project

### PR DESCRIPTION
**Pull request by Fatme**
http://teampulse.telerik.com/view#item/290014

### CODE REVIEW

File: lib/commands/project/project-command-base.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+export class ProjectCommandBase {
```
Comment (rosen-vladimirov) [inline lib/commands/project/project-command-base.ts]: why do we need this base class, can't we just add canExecute method of the command

**Comment by teobugslayer** [comment]
:+1:

File: lib/commands/project/project-command-base.ts (hunk 1 above)
Comment (Fatme) [inline lib/commands/project/project-command-base.ts]: Maybe we can use `canExectute`, but I need to copy paste this check in `create.ts` and in `create-project.ts`.  I need it to be only on one place.

Comment (rosen-vladimirov) [inline lib/commands/project/project-command-base.ts]: aaah, correct

**Comment by rosen-vladimirov** [comment]
:+1:

File: lib/commands/project/project-command-base.ts (hunk 1 above)
Comment (Fatme) [inline lib/commands/project/project-command-base.ts]: Actually, this check must be in `canExecute`. I realize it after the failed QA tests :smile_cat:

**Pull request merged**